### PR TITLE
gitlab: publish install-mender.sh to s3

### PR DESCRIPTION
**Pull request by mchalski**
one missing acc criterion for https://tracker.mender.io/browse/MEN-4150.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -8,6 +8,8 @@ variables:
   S3_BUCKET_SUBPATH: "dist-packages/debian"
   # APT repo path
   S3_BUCKET_REPO_PATH: "repos/debian"
+  # Scripts folder subpath
+  S3_BUCKET_SCRIPTS_PATH: "scripts"
```
Comment (lluiscampos) [inline .gitlab-ci.yml:12]: I've been inspecting a bit the bucket. We have a lot of stuff here, and I think is better for the script to share the directory with the repos. Let's change this to
```suggestion
  # Scripts folder subpath
  S3_BUCKET_SCRIPTS_PATH: "repos/scripts"
```

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -166,3 +168,19 @@ publish:s3:legacy:mender-client:latest:
...
+publish:s3:scripts:install-mender-sh:
+  stage: publish
+  dependencies:
+    - test:acceptance
+  image: debian:buster
+  before_script:
+    - apt update && apt install -yyq awscli
+  script:
+    - echo "Publishing install-mender.sh to S3"
+    - aws s3 cp scripts/install-mender.sh s3://$S3_BUCKET_NAME/$S3_BUCKET_SCRIPTS_PATH/install-mender.sh
```
Comment (lluiscampos) [inline .gitlab-ci.yml:181]: We need  to set public access explicitly, see other publish jobs above.
```suggestion
    - aws s3 cp scripts/install-mender.sh s3://$S3_BUCKET_NAME/$S3_BUCKET_SCRIPTS_PATH/install-mender.sh
    - aws s3api put-object-acl --acl public-read --bucket ${S3_BUCKET_NAME}
          --key latest/${S3_BUCKET_SCRIPTS_PATH}/install-mender.sh
```

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Thanks for fixing and sorry I missed it in the first PR.

**Comment by mchalski** [comment]
@lluiscampos fixed

**Comment by lluiscampos** [comment]
> @lluiscampos fixed

@mchalski Still missing the `put-object-acl` call, see https://github.com/mendersoftware/mender-dist-packages/pull/50#discussion_r533345927

**Comment by mchalski** [comment]
> > @lluiscampos fixed
> 
> @mchalski Still missing the `put-object-acl` call, see [#50 (comment)](https://github.com/mendersoftware/mender-dist-packages/pull/50#discussion_r533345927)

@lluiscampos sorry, missing amend...now?

**Review by lluiscampos — APPROVED** [review]

**Comment by lluiscampos** [comment]
Merging; these changes do not affect PR build.

**Pull request merged**